### ISSUE TITLE
Engine start

### ISSUE THREAD

**Issue by mrshiljo** [flybywiresim/a32nx#8050]
### Aircraft Version

Experimental

### Build info

```json
...
```

### Describe the bug

Well, as stated, on FBW when starting engines with fuel pumps, nothing hapoens. Thats not correct, if u start engine, starter will run normally untill  en g it reaches its max on N2, regardless fuel pump are off. Ok, fuel is not present, eng will not start, it wont get fuel,but it will reach some parameters because starter is working. 

### Expected behavior

Eng must rotate because starter is working

### Steps to reproduce

...

**Comment by Luz5020** [comment]
Last time I tried, the Engine parameters definitely went up as far as they can go on starter only, is the issue that the engine on the model doesn't start to spin?

**Comment by Maximilian-Reuter** [comment]
To be precise, do you mean there is an issue with the animation of the engine ? Or are you talking about the fact that gravity feeding doesnt work, cause if so then this is a duplicate of #2841

**Comment by mrshiljo** [comment]
Noooo...i started engine by fault with fuel pumps off, and it didnt start,
N2 was stuck below 1...so...when you push fuel pump pushbutton
additionaly...engine starts to move. You have it on my video on 16:51

https://www.youtube.com/live/Z3mwSzFor5s?feature=share

So let me repeat...engine will start, and it will even run for a while in
rl  ecause some of fuel is left in engine system. I did manage to do that
while starting once for troubleshooting purposes

Kind regards




pon, 29. svi 2023. 19:27 Chaoz ***@***.***> je napisao:

**Comment by mrshiljo** [comment]
So i dont men manualy starting by starter, i mean nirmal starting automatic
without fuel pumps. On FBW engine dont move, dont rotate, N2 is 0,  but it
should go up to at least 27

pon, 29. svi 2023. 23:17 Shiljo ***@***.***> je napisao:

**Comment by hotshotp** [comment]
> I now understand, the issue is that the engines arent even motoring without the fuelpumps.
> 
> That is indeed an issue.
> 
> EDIT: In my tests the engines do **very** slowly motor up to 27 %N2 when no fuelpumps are on. Even if you turn them on at that point it starts up **very** slowly. This is probably an issue with the MSFS Engine model as the FBW engine parameters still depend on the msfs engine parameters

they dont depend on the msfs engine parameters, we have a completely custom engine model outside the sim. i think @Taz5150 would know better.

**Comment by tracernz** [comment]
> they dont depend on the msfs engine parameters

They definitely do to an extent. Basically the FBW code reads the sim parameters, then fudges some numbers and writes back to the sim some parameters.

**Comment by Maximilian-Reuter** [comment]
> > I now understand, the issue is that the engines arent even motoring without the fuelpumps.
> > That is indeed an issue.
> > EDIT: In my tests the engines do **very** slowly motor up to 27 %N2 when no fuelpumps are on. Even if you turn them on at that point it starts up **very** slowly. This is probably an issue with the MSFS Engine model as the FBW engine parameters still depend on the msfs engine parameters
> 
> they dont depend on the msfs engine parameters, we have a completely custom engine model outside the sim. i think @Taz5150 would know better about this issue than me.

the entire system is dependant on the msfs N2 so unless that changes our N2 doesnt either. Hence why it starts up much much slower when it doesnt have the fuel pumps as the MSFS engine model doesnt start without fuelpumps.

**Comment by Taz5150** [comment]
Sorry I am late to this conversation, but basically you all nailed it. Our model is external mostly to control engine spool times during startup & shutdown, but it does depend on N2. So if using SimVarWatcher you observe N2 rising, then this is an MSFS issue.

Now that being said, I will investigate a little further to understand why is that so.
